.gitignore

```diff
@@ -9,4 +9,5 @@
 
 build
 .vscode
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vs
```

Comment:
Any reason for ignoring this folder?

Reply:
This is added whenever you open a folder in full Visual Studio.  Excluded it since it isn't needed.